compute/gpu/concepts.mdx

```diff
@@ -16,22 +16,102 @@ categories:
 
 <Concept opened>
 
-## Docker
+## AI-Enabled applications
```

Comment:
```suggestion
## AI-Enabled Applications
```

Reply:
@jsudraud If **AI-Enabled applications** is not the name of a product/feature, I'd keep it lowercase.

Reply:
Yes, you're right. I thought it was a title, but it's not